sigal/gallery.py

```diff
@@ -186,8 +190,8 @@ class Video(Media):
     def __init__(self, filename, path, settings):
         super(Video, self).__init__(filename, path, settings)
         base, ext = splitext(filename)
-        self.date = None
         self.src_filename = filename
+        self.date = self._get_file_date() or None
```

Comment:
`_get_file_date` should always return a date, so the `or None` seems useless (and the same above).